test/threads_test.py

```diff
@@ -25,14 +25,31 @@ def f2(x):
         self.assertEqual(fr.result, 2)
         self.assertEqual(fr2.result, 3)
 
-    def todo_test_do(self):
+    def test_do(self):
 
         # __doc__ (as of 2008-06-28) for pygame.threads.WorkerQueue.do:
 
-        # puts a function on a queue for running later.
+        # puts a function on a queue for running _later_.
         #
+        def sleepTest():
+            time.sleep(1)
 
-        self.fail()
+        def calcTest(x):
+            return x + 1
+
+        wq = WorkerQueue(num_workers=1)
+        sleepReturn = FuncResult(sleepTest)
+        calcReturn = FuncResult(calcTest)
```

Comment:
more camelCase on these two variable names. camelCase is only allowed on Class names under the strictures of PEP8 for whatever reason.

Reply:
No worries. I will fix these for PEP8.